src/NUnitConsole/nunit3-console/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This probably shouldn't be included. Maybe we should add to `.gitignore`?

Reply:
Yeah, I thought that. This is just for CI builds for test at the moment...but I’ll tidy up if it ever comes to anything. 😊